docs/pipeline/enrichments/available-enrichments/custom-javascript-enrichment/writing/index.md

```diff
@@ -328,3 +328,18 @@ function process(event, params, headers) {
   }
 }
 ```
+
+## Erasing derived contexts
```

Comment:
Could you move this to right after line 242 (which talks about modifying entities/contexts)? Also there is a note there about `setDerived_contexts` not being a thing. Could you update that note to refer to this section as the alternative?